jenkins/Jenkinsfile

```
@@ -31,6 +31,30 @@ pipeline {
             '''
           }
         }
+        stage('NDTensorsCUDAExt julia-1.11') {
+	        options {
+            timeout(time: 45, unit: 'MINUTES')
+          }
+          agent {
+            dockerfile {
+              label 'gpu&&v100'
+              filename 'Dockerfile'
+              dir 'jenkins'
+              additionalBuildArgs  '--build-arg JULIA=1.11'
```

Similar to the other comments, hopefully we can change this to something like:
```suggestion
              additionalBuildArgs  '--build-arg JULIA=1'
```
i.e. don't hard code `1.11` if we don't have to, though I don't know enough about Jenkins build scripts to understand what this line is doing. Is `JULIA=1.11` actually setting the Julia version, or is this used more for printing?

---

The `JULIA=#` arg is being used to apt-get the correct julia version. I think that `1` should work and just keep the latest 1.x release. I'll push the change and we will find out!

---

Gotcha, can it be rewritten to use `juliaup` instead of `apt-get`? That is better supported and I see `juliaup` being used in other parts of the workflow.

---

Looks like maybe `apt-get` doesn't expand `JULIA=1` to the latest stable release (`JULIA=1.11`).

---

Let me know if it makes sense and wouldn't be too hard to switch over from `apt-get` to `juliaup`, otherwise having this one spot hard coded to 1.11 isn't too bad and we could look into improving that (say with `juliaup`) later.